.github/workflows/pr-to-update-winget.yml

```diff
@@ -0,0 +1,129 @@
+name: PR for updating winget
+
+on:
+  release:
+    types: [released]
+
+jobs:
+  winget-update:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout winget-automation
+        uses: actions/checkout@v4
+        with:
+          ref: winget-automation
```

Comment:
Why do we need this ref?

Reply:
I followed the docs to match the branch name so I can test locally when doing practice runs before learning about doing it from a fork, I think this wont be needed for the main structure